src/components/documents/CmsText/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { observer } from 'mobx-react-lite';
+import { CmsTextContext, useFirstCmsTextDocumentIfExists } from '@tdev-components/documents/CmsText/shared';
+import React from 'react';
+
+interface Props {
+    id?: string;
+}
+
+const CmsText = observer(({ id }: Props) => {
+    const context = React.useContext(CmsTextContext);
+
+    let cmsText: string | undefined;
+
+    if (context && !id) {
+        cmsText = context.cmsText;
+    } else if (!context && id) {
+        cmsText = useFirstCmsTextDocumentIfExists(id)?.text;
+    } else {
+        throw new Error('Either provide an id property or use inside <WithCmsText> (but not both)');
```

Comment:
i wonder if this is not too restrictive? Couldn't the following usecase be handy? 

```mdx
<WithCmsText id="xxx">
    <DefinitionList>
        <dt>Punkte</dt>
        <dd><CmsText /></dd>
        <dt>Note</dt>
        <dd><CmsText id="yyy" /></dd>
        <dt>Kommentar</dt>
        <dd><CmsText id="zzz" /></dd>
    </DefinitionList>
</WithCmsText>
```

But yah, it doesn't hurt to write it more explicitely...

Reply:
Yes, this makes sense. I'm also thinking about changing the `<WithCmsText>`-API to something like this:

```mdx
<WithCmsText entries={{punkte: "xxx", note: "yyy"}}>
    <DefinitionList>
        <dt>Punkte</dt>
        <dd><CmsText name="punkte" /></dd>
        <dt>Note</dt>
        <dd><CmsText name="note" /></dd>
        <dt>Kommentar</dt>
        <dd><CmsText id="zzz" /></dd>
    </DefinitionList>
</WithCmsText>
```

That way, we would have the following behavior:
- Unless there is a document present for both root IDs `xxx` and `yyy`, the entire DefinitionList is hidden.
- If only the one for `zzz` is missing, there's just an empty `<dd>`-entry for `<dt>Kommentar</dt>`.

Using such a map is technically an "unnecessary" indirection, but I think it would make the whole thing a bit more readable (as opposed to having to write UUIDs everywhere).

What do you think?